src/lib.rs

```diff
@@ -87,20 +89,25 @@ pub mod vc {
     pub use super::iter::*;
 }
 
+#[cfg(any(test, miri))]
+pub type Vc<T> = RawVc<T, 4>;
+#[cfg(not(any(test, miri)))]
+pub type Vc<T> = RawVc<T, 8>;
```

Comment:
Thinking about this some more, let's do:
```rust
const fn default_r() -> usize {
    #[cfg....]
    4
}

pub type Vc<T> = RawVc<T, { default_r() }>;

pub struct RawVc<T, const R: usize = { default_r() }> {
...
}
```
That way people can still make use of the default `R` when using `RawVc` directly.

Reply:
As we seem to support `1.52.1` (judging by the check in the CI).
We cant use this pattern:
```
pub struct RawVc<T, const R: usize = { default_r() }> {
...
}
```
as it fails with `error[E0658]: default values for const generic parameters are experimental`

Reply:
I'd be okay bumping the MSRV for this :+1:

Reply:
Alright, is the MSRV documented anywhere?

Reply:
No, it's just checked in CI and not mentioned anywhere I believe.